src/app/content/components/Toolbar/__snapshots__/PracticeQuestions.spec.tsx.snap

```diff
@@ -2,21 +2,4 @@
 
 exports[`practice questions button does not render if feature flag is not enabled 1`] = `null`;
 
-exports[`practice questions button render if feature flag is enabled 1`] = `
-<button
-  aria-label="Practice"
-  className="styled__PlainButton-sc-1119auh-0 PracticeQuestionsButton__PracticeQuestionsWrapper-vyu0p6-0 dXEzXZ"
-  onClick={[Function]}
->
-  <img
-    aria-hidden="true"
-    className="PracticeQuestionsButton__PracticeQuestionsIcon-vyu0p6-1 jjDwhv"
-    src="practiceQuestionsIcon.svg"
-  />
-  <span
-    className="PracticeQuestionsButton__PracticeQuestionsText-vyu0p6-2 fJxtws"
-  >
-    Practice
-  </span>
-</button>
-`;
+exports[`practice questions button render if feature flag is enabled 1`] = `null`;
```

Comment:
Please update this test name to `practice questions button render if feature flag is enabled and there are practice questions` and also add test `practice questions button does not render if feature flag is enabled but there are no practice questions`